apps/src/lib/kits/maker/boards/microBit/LightSensor.js

```diff
@@ -67,11 +82,39 @@ export default class LightSensor extends EventEmitter {
     this.board.mb.stopStreamingAnalogChannel(sensor_channels.lightSensor); // disable light sensor
   }
 
-  //TODO
-  getAveragedValue() {}
+  // Get the averaged value over the give range of ms, adjusted within specified range
+  // TODO: Handle if the range is outside of 50 (before first sample) or 3000 (longer than our buffer).
+  // Currently returns null if range is outside of acceptable values
+  getAveragedValue(range) {
```

Comment:
A "range" seems to imply a start and end point, but I think here we're really specifying a duration, which happens to end at the current moment..